EIPS/eip-2718.md

```diff
@@ -1,6 +1,7 @@
 ---
 eip: 2718
 title: Typed Transaction Envelope
+description: Defines a new transaction type that is an envelope for future transaction types.
```

Comment:
Simple Summary should be removed with the inclusion of description. I think.

Reply:
Right. My question was do you want to touch this file in this PR or not?

Reply:
I'm fine with 2718 getting updated as a "first example" in this PR.

Reply:
Ok, also removed the section.